### DISCUSSION TITLE
拼音设置下的[词典]是词库吗?什么格式的?

### DISCUSSION THREAD

**Discussion by sunwgit** [fcitx5-android/fcitx5-android#167] [Q&A | 问答] [answered]
rt

**Comment by rocka** [comment, marked as answer]
是的。可以使用三种格式：

- 搜狗的[细胞词库](https://pinyin.sogou.com/dict/)，拓展名为 `.scel` 。应用已经注册了 `.scel` 拓展名的打开方式，一般直接在文件管理器中打开即可导入。
- libime 的 `.txt` 格式，每行一个词，按照
	```
	拼音	pin'yin	0
	词条	全拼		权重
	```
	的方式排列。具体例子可以参考[内置的 emoji 词典](https://github.com/fcitx/fcitx5-chinese-addons/blob/5.0.16/im/pinyin/emoji.txt)。
- libime 的 `.dict` 格式。使用命令行工具 `libime_pinyindict` 从上面的 `.txt` 格式转换而来。实际上导入 `.txt` 格式的词库时，会自动转换成 `.dict` 格式，一般一些第三方词库会提供这种格式，比如 https://github.com/felixonmars/fcitx5-pinyin-zhwiki 和 https://github.com/outloudvi/mw2fcitx 。

如果需要自己制作词库，使用 `.txt` 格式即可。
